rental.rb

```diff
@@ -0,0 +1,13 @@
+class Rental
+  attr_accessor :date
+  attr_reader :book, :person
```

Comment:
- [ ] The :book and :person are supposed to have both the getter and setters set for them. Currently, I can see that only the getters are set for them. Kindly add the setter as well to it. This will make it look like this: ```attr_accessor :date, :book, :person```

Reply:
Fixed.